new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleTraefikValidation.java

```diff
@@ -0,0 +1,678 @@
+   * operator namespace.
+   */
+  @AfterAll
+  public void tearDownAll() {
+    // Delete domain custom resource
+    logger.info("Delete domain custom resource in namespace {0}", domainNamespace);
+    assertDoesNotThrow(() -> deleteDomainCustomResource(domainUid, domainNamespace),
+        "deleteDomainCustomResource failed with ApiException");
+    logger.info("Deleted Domain Custom Resource " + domainUid + " from " + domainNamespace);
+
+    // delete the domain image created for the test
+    if (miiImage != null) {
+      deleteImage(miiImage);
+    }
+
+    // uninstall operator release
+    logger.info("Uninstall operator in namespace {0}", opNamespace);
+    if (opHelmParams != null) {
+      uninstallOperator(opHelmParams);
+    }
+    // Delete service account from unique opNamespace
+    logger.info("Delete service account in namespace {0}", opNamespace);
+    if (serviceAccount != null) {
+      assertDoesNotThrow(() -> deleteServiceAccount(serviceAccount.getMetadata().getName(),
+          serviceAccount.getMetadata().getNamespace()),
+          "deleteServiceAccount failed with ApiException");
+    }
+    // Delete domain namespaces
+    logger.info("Deleting domain namespace {0}", domainNamespace);
+    if (domainNamespace != null) {
+      assertDoesNotThrow(() -> deleteNamespace(domainNamespace),
+          "deleteNamespace failed with ApiException");
+      logger.info("Deleted namespace: " + domainNamespace);
+    }
+
+    // Delete opNamespace
+    logger.info("Deleting operator namespace {0}", opNamespace);
+    if (opNamespace != null) {
+      assertDoesNotThrow(() -> deleteNamespace(opNamespace),
+          "deleteNamespace failed with ApiException");
+      logger.info("Deleted namespace: " + opNamespace);
+    }
+
+    // uninstall ingress
+    if (ingressParam != null) {
+      assertThat(uninstallIngress(ingressParam))
+          .as("Test uninstallIngress returns true")
+          .withFailMessage("uninstallIngress() did not return true")
+          .isTrue();
+    }
+
+    // uninstall Traefik release
+    if (tfHelmParams != null) {
+      assertThat(uninstallTraefik(tfHelmParams))
+          .as("Test uninstallTraefik returns true")
+          .withFailMessage("uninstallTraefik() did not return true")
+          .isTrue();
+    }
+
+    // Delete Traefik Namespace
+    if (tfNamespace != null) {
+      assertThatCode(
+          () -> deleteNamespace(tfNamespace))
+          .as("Test that deleteNamespace does not throw an exception")
+          .withFailMessage("deleteNamespace() threw an exception")
+          .doesNotThrowAnyException();
+      logger.info("Deleted namespace: {0}", tfNamespace);
+    }
+  }
+
+  /**
+   * Prepare and install WebLogic operator.
+   */
+  private static void installAndVerifyOperator() {
+    // get a new unique opNamespace
+    logger.info("Creating unique namespace for operator");
+    opNamespace = assertDoesNotThrow(() -> createUniqueNamespace(),
```

Comment:
namespaces should not be created in test class, they are created in the extension class IntegrationTestWatcher, take a look at ItMiiDomain.java and change the code accordingly.